crates/sui-core/src/epoch/randomness.rs

```diff
@@ -243,12 +337,12 @@ impl RandomnessManager {
             // Load intermediate data.
             rm.processed_messages.extend(
                 tables
-                    .dkg_processed_messages
+                    .dkg_versioned_processed_messages
```

Comment:
doesn't this still need to load from the old tables if the protocol config has dkg version set to 0?

Reply:
once we run the new binary, we use the new tables for both versions.
that said, there could be a problem if the binary is replaced during the dkg, as some data is stored in the old tables while after binary upgrade, the validator is using the new ones.
added code to load from the old tables in that case as well.